商品一覧表示機能

Yasuhisa-O opened the pull request:
# What
　テーブルに保存された商品をメイン画面に表示する
# Why
　itemsコントローラ内のindexアクションを記述、eachメソッドを使って商品情報を表示した。

＜商品のデータがない場合は、ダミー商品が表示されている動画＞
https://gyazo.com/a1623d3539ee78e67321aa09632364d1

＜商品のデータがある場合は、商品が一覧で表示されている動画＞
https://gyazo.com/acc99a2e3d9fa0c8234bf9ea5308ca78

Yasuhisa-O committed:
出品商品の表示

Yasuhisa-O committed:
orderモデル作成、ダミー表示の切り替え機能を追加

Yasuhisa-O committed:
soldoutの表示機能を追加、ダミー表示の方法を見直し

Yasuhisa-O committed:
不要なコメントを削除

mihot117 commented on app/controllers/items_controller.rb:
```diff
@@ -2,6 +2,7 @@ class ItemsController < ApplicationController
   before_action :authenticate_user!,  only: [:new]
 
   def index
+    @items = Item.all
   end
```
出品された商品は降順で表示されるように修正しましょう！

■理由 
 以下の実装条件を満たすためです：
左上から、出品された日時が新しい順に表示されること。

Yasuhisa-O replied:
ご確認と理由の記入、
有難うございます。
”@items = Item.all.order(created_at: :desc)”
で降順表示に変更します。

mihot117 reviewed (commented):
レビュー依頼ありがとうございます。

コメントいたしましたので、ご確認をお願いいたします。

コードの修正後、コミット・プッシュを行った上で再度レビュー依頼をお送りください。
ご対応のほど、よろしくお願いいたします。

Yasuhisa-O committed:
降順表示に変更

＜商品のデータがない場合は、ダミー商品が表示されている動画＞
https://gyazo.com/a1623d3539ee78e67321aa09632364d1

＜商品のデータがある場合は、商品が一覧で表示されている動画＞
https://gyazo.com/c666c91842e9dc3a5cda3e1a7d486e17

TAKADA-yuma reviewed (commented):
レビュー依頼ありがとうございます！
LGTMです！
商品一覧表示機能の実装、お疲れ様でした！

この後の流れの確認です。
①Git上でマージ
②ローカルのmasterブランチへ切り替え
③Fetch Originしプル

そのあと、新しいブランチを作成し、次の実装に取り組んでください。
引き続き、実装頑張っていきましょう。
応援しております！
